samples/bluetooth/llpm/README.rst

```diff
@@ -106,33 +106,29 @@ After programming the sample to both development kits, test it by performing the
 
    - The central outputs the following information::
 
-       Press any key to set LLPM short connection connection interval (1 ms)
+       Press any key to start measuring transmission latency
 
    - The peripheral outputs the following information::
 
        Press any key to start measuring transmission latency
 
 #. Press a key in the terminal that is connected to the peripheral.
 #. Observe the terminal connected to the peripheral. The latency measurements are printed in the terminal.
-   The latency is expected to be shorter than the default connection interval::
+   The latency is expected to be around 1ms.
 
-       Transmission Latency: 80917 (us), CRC mismatches: 0
+       Transmission Latency: 1083 (us), CRC mismatches: 0
 
 #. Press a key in the terminal that is connected to the central.
-#. Observe the connection gets updated to LLPM connection interval (1 ms) on both sides::
-
-       Connection interval updated: LLPM (1 ms)
-
 #. Observe the terminal connected to the peripheral.
    The measured latency on the peripheral becomes approximate 1 ms::
 
        Transmission Latency: 1098 (us), CRC mismatches: 0
 
-#. Press a key in the terminal that is connected to the central.
-#. Observe the terminal connected to the central.
-   The measured latency on the central remains approximate 1 ms::
+#. After the central has done a few measurments it switches to standard 7.5ms interval
```

Comment:
1. Is the second part of this sentence dependent on the results of these measurements? Or does it happen always after doing measurements? What kind of measurements are done here? "A few measurements" is too vague.

2. The first sentence in each step should contain only actions for end users, hence this sentence should be rephrased. Is this something that can be observed/checked/verified by end user?
For example:
```suggestion
#. Observe the central switches to standard 7.5 ms interval right after performing latency measurements.

```

Reply:
it is hardcoded to 20, but I didn't want to write that in the doc so that we have to worry about keeping doc and implementation in sync. The switch is not dependent on anything no. It does it regardless of meansurment results.